manifests/istio-control/istio-discovery/templates/configmap.yaml

```diff
@@ -254,16 +254,16 @@ data:
       {{- end }}
 
     {{- if not (eq .Values.revision "") }}
-    {{- $defPilotHostname := printf "istiod-%s.%s" .Values.revision .Release.Namespace }}
+    {{- $defPilotHostname := printf "istiod-%s.%s.svc" .Values.revision .Release.Namespace }}
     {{- else }}
-    {{- $defPilotHostname := printf "istiod.%s"  .Release.Namespace }}
+    {{- $defPilotHostname := printf "istiod.%s.svc" .Release.Namespace }}
     {{- end }}
-    {{- $defPilotHostname := printf "istiod%s.%s" .Values.revision .Release.Namespace }}
+    {{- $defPilotHostname := printf "istiod%s.%s.svc" .Values.revision .Release.Namespace }}
```

Comment:
Doesn't line 261 overwrite 257/269?

Reply:
I think line 261 covers a special case - probably when .Values.revision is not specified at all.

I had initially removed line 261 because I didn't understand what it is doing but I was getting some errors at make gen.

@howardjohn comments?  Think you created this line before.